includes/cli-aliases.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This is in two places. Should we consolidate a bit? Also, the discussion in the CLI design guidelines has more information.